CRITICAL - 2025-02-09 08:59:31 --> Invalid file: v_template. $data
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_template. $data')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template. $data', [], true)
 3 APPPATH\Controllers\Home.php(12): view('v_template. $data')
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 08:59:40 --> Invalid file: v_template. $data
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_template. $data')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template. $data', [], true)
 3 APPPATH\Controllers\Home.php(12): view('v_template. $data')
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 09:00:09 --> syntax error, unexpected variable "$data", expecting ")"
in APPPATH\Controllers\Home.php on line 12.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(848): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 09:06:08 --> Invalid file: v_home.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_home.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_home', [], true)
 3 APPPATH\Views\v_template.php(207): view('v_home')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 7 APPPATH\Controllers\Home.php(12): view('v_template', [...])
 8 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Home->index()
 9 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 16:48:37 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 219.
 1 SYSTEMPATH\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-09 17:22:27 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 219.
 1 SYSTEMPATH\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-09 20:58:40 --> Undefined variable $menu
in APPPATH\Views\v_template.php on line 84.
 1 APPPATH\Views\v_template.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $menu', 'APPPATH\\Views\\v_template.php', 84)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 5 APPPATH\Controllers\Part.php(14): view('v_template', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Part->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Part))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 21:09:22 --> Invalid file: v_satuan.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_satuan.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_satuan', [], true)
 3 APPPATH\Views\v_template.php(193): view('v_satuan')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 7 APPPATH\Controllers\Satuan.php(18): view('v_template', [...])
 8 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 9 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
10 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 21:10:04 --> Invalid file: v_satuan.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_satuan.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_satuan', [], true)
 3 APPPATH\Views\v_template.php(193): view('v_satuan')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 7 APPPATH\Controllers\Satuan.php(18): view('v_template', [...])
 8 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 9 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
10 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 21:10:07 --> Invalid file: v_satuan.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_satuan.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_satuan', [], true)
 3 APPPATH\Views\v_template.php(193): view('v_satuan')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 7 APPPATH\Controllers\Satuan.php(18): view('v_template', [...])
 8 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 9 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
10 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 21:10:09 --> Undefined variable $menu
in APPPATH\Views\v_template.php on line 84.
 1 APPPATH\Views\v_template.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $menu', 'APPPATH\\Views\\v_template.php', 84)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 5 APPPATH\Controllers\User.php(14): view('v_template', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\User->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 21:11:56 --> Undefined variable $menu
in APPPATH\Views\v_template.php on line 84.
 1 APPPATH\Views\v_template.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $menu', 'APPPATH\\Views\\v_template.php', 84)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 5 APPPATH\Controllers\User.php(14): view('v_template', [...])
 6 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\User->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:55:41 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:55:41 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:57:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:57:47 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:57:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:57:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:59:37 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:59:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:59:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:59:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:59:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:59:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 22:59:57 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 22:59:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2025-02-09 23:01:25 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'posbengkel', 3306, '', 0)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\posbengkel\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\posbengkel\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\posbengkel\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\posbengkel\app\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\posbengkel\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(898): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\posbengkel\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\posbengkel\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\posbengkel\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2025-02-09 23:01:25 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `satuan`', [], false)
 3 APPPATH\Models\ModelSatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 23:32:27 --> Undefined variable $no
in APPPATH\Views\v_satuan.php on line 26.
 1 APPPATH\Views\v_satuan.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $no', 'APPPATH\\Views\\v_satuan.php', 26)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_satuan.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_satuan', [], true)
 5 APPPATH\Views\v_template.php(198): view('v_satuan')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 9 APPPATH\Controllers\Satuan.php(25): view('v_template', [...])
10 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
11 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
12 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 23:32:34 --> Undefined variable $no
in APPPATH\Views\v_satuan.php on line 26.
 1 APPPATH\Views\v_satuan.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $no', 'APPPATH\\Views\\v_satuan.php', 26)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_satuan.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_satuan', [], true)
 5 APPPATH\Views\v_template.php(198): view('v_satuan')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 9 APPPATH\Controllers\Satuan.php(25): view('v_template', [...])
10 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
11 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
12 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2025-02-09 23:35:20 --> Array to string conversion
in APPPATH\Views\v_satuan.php on line 30.
 1 APPPATH\Views\v_satuan.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Views\\v_satuan.php', 30)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_satuan.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_satuan', [], true)
 5 APPPATH\Views\v_template.php(198): view('v_satuan')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template', [], true)
 9 APPPATH\Controllers\Satuan.php(25): view('v_template', [...])
10 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\Satuan->index()
11 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
12 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
